tox.ini

```diff
@@ -4,18 +4,30 @@
 # and then run "tox" from this directory.to
 
 [tox]
-envlist = py37, docs
+envlist = py, docs
 isolated_build = True
 
 [testenv]
 extras =
     test
 commands =
-    pytest
+    pytest test/test_dummy.py -v
```

Comment:
We should not test single files but rather all files within a directory. This means that the test directory should get subdirs like
test
       base
       torch
       tensorflow

Reply:
Done